sale_order_line_copy/__manifest__.py

```diff
@@ -0,0 +1,18 @@
+# © 2025 Solvos Consultoría Informática (<http://www.solvos.es>)
+# License LGPL-3 - See http://www.gnu.org/licenses/lgpl-3.0.html
+{
+    "name": "Sale Order Line Copy",
+    "summary": """
+        Adds an icon to copy sales order lines
```

Comment:
Complete info (make it also in `readme` file):
```suggestion
        Adds an icon to copy sales order lines within sales order form
```